charts/tfy-buildkitd-service/templates/_helpers.tpl

```diff
@@ -196,4 +196,30 @@ limits:
 {{- else if eq $tier "large" -}}
 200Gi
 {{- end }}
+{{- end }}
+
+{{/*
+Affinity for the buildkitd service
+*/}}
+{{- define "buildkitd-service.affinity" -}}
+{{- if .Values.affinity -}}
+{{ toYaml .Values.affinity }}
```

Comment:
If we are using `nindent` in the main template file, should we make it
```
{{- toYaml
```

Reply:
If yes, we should make the change everywhere

Reply:
is there a significant difference or advantage to doing this? I already tested the output and no extra lines/spaces were created